Echo.Process/Session/SessionSync.cs

```diff
@@ -70,7 +70,7 @@ public int Incoming(SessionAction incoming)
                     Stop(incoming.SessionId);
                     break;
                 case SessionActionTag.SetData:
-                    var type = Type.GetType(incoming.Type);
+                    var type = typeof(object);//Type.GetType(incoming.Type);
```

Comment:
This doesn't look pretty, I think I'll call you this morning to discuss the serialisation issues you're having.